superset/utils/pandas_postprocessing/sort.py

```diff
@@ -14,22 +14,34 @@
 # KIND, either express or implied.  See the License for the
 # specific language governing permissions and limitations
 # under the License.
-from typing import Dict
+from typing import List, Optional, Union
 
 from pandas import DataFrame
 
 from superset.utils.pandas_postprocessing.utils import validate_column_args
 
 
-@validate_column_args("columns")
-def sort(df: DataFrame, columns: Dict[str, bool]) -> DataFrame:
+# pylint: disable=invalid-name
+@validate_column_args("by")
+def sort(
```

Comment:
This function is useless as before so redesigning the interface is safe. The interface refers to Pandas.